content/src/migrations/scripts/1620070748842_remove_origin.ts

```diff
@@ -0,0 +1,24 @@
+import { MigrationBuilder } from 'node-pg-migrate'
+
+/*
+  In this migration, we are removing the origin_server_url and origin_timestamp colums
+ */
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.sql(`DROP INDEX IF EXISTS deployments_origin_timestamp_idx;`)
```

Comment:
Can you check if this is necessary or just removing the column is enough?

Reply:
You're right, according to the documentation https://www.postgresql.org/docs/9.1/sql-altertable.html the drop table removes all indexes associated with the column, will remove the statement